specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/examples/streamingendpoint-update.json

```diff
@@ -6,7 +6,6 @@
     "streamingEndpointName": "myStreamingEndpoint1",
     "api-version": "2020-05-01",
     "parameters": {
-      "id": null,
       "location": "West US",
       "tags": {
         "tag3": "value3",
```

Comment:
For all of our example tags, should we use a real example? Like "createdBy" or "Owner", or "ExampleTenantName" as an example?

Reply:
lets try to get these examples update for Nov release. I can work with Ning and you to update the examples.